CollAction/Views/Home/FAQ.cshtml

```diff
@@ -281,5 +281,20 @@
             </div>
         </div>
     </div>
+
+    <div class="panel panel-default">
+        <div class="panel-heading" data-toggle="collapse" href="#collapseHowKnow">
+            <h3 class="panel-title">
+                <span class="fa fa-rocket" aria-hidden="true"></span>&nbsp;
+                When will the projets end exactly?
```

Comment:
This has to be translated to dutch.

It's: "Wanneer eindigen de projecten precies?"

Also, I think you misspelled "projets" in the original. Might need another pull request.